src/test/e2e/bundle_test.go

```diff
@@ -108,138 +110,143 @@ func TestBundle(t *testing.T) {
+	createLocal(t, bundleDir, e2e.Arch)
 	inspectLocal(t, bundlePath)
 	inspectLocalAndSBOMExtract(t, bundlePath)
-	// Test with an "options only" config file
-	os.Setenv("UDS_CONFIG", filepath.Join("src/test/bundles/01-uds-bundle", "uds-config.yaml"))
 	deploy(t, bundlePath)
 	remove(t, bundlePath)
-
-	//Test create using custom tmpDir
-	runCmd(t, "create "+bundleDir+" --tmpdir ./customtmp --confirm --insecure")
-
-	// remove customtmp folder if it exists
-	err := os.RemoveAll("./customtmp")
-	require.NoError(t, err)
-
 }
 
 func TestPackagesFlag(t *testing.T) {
 	deployZarfInit(t)
-
-	e2e.CreateZarfPkg(t, "src/test/packages/nginx", false)
 	e2e.CreateZarfPkg(t, "src/test/packages/podinfo", false)
-
-	e2e.SetupDockerRegistry(t, 888)
-	defer e2e.TeardownRegistry(t, 888)
-	e2e.SetupDockerRegistry(t, 889)
-	defer e2e.TeardownRegistry(t, 889)
-
-	pkg := fmt.Sprintf("src/test/packages/nginx/zarf-package-nginx-%s-0.0.1.tar.zst", e2e.Arch)
-	zarfPublish(t, pkg, "localhost:888")
-
-	pkg = fmt.Sprintf("src/test/packages/podinfo/zarf-package-podinfo-%s-0.0.1.tar.zst", e2e.Arch)
-	zarfPublish(t, pkg, "localhost:889")
-
-	bundleDir := "src/test/bundles/01-uds-bundle"
-	bundlePath := filepath.Join(bundleDir, fmt.Sprintf("uds-bundle-example-%s-0.0.1.tar.zst", e2e.Arch))
+	bundleDir := "src/test/bundles/03-local-and-remote"
+	bundlePath := filepath.Join(bundleDir, fmt.Sprintf("uds-bundle-test-local-and-remote-%s-0.0.1.tar.zst", e2e.Arch))
 
 	createLocal(t, bundleDir, e2e.Arch)
-	inspectLocal(t, bundlePath)
-	inspectLocalAndSBOMExtract(t, bundlePath)
-
-	// Test only podinfo deploy
-	deployPackagesFlag(bundlePath, "podinfo")
-	cmd := strings.Split("zarf tools kubectl get deployments -A -o=jsonpath='{.items[*].metadata.name}'", " ")
-	deployments, _, _ := e2e.UDS(cmd...)
-	require.Contains(t, deployments, "podinfo")
-	require.NotContains(t, deployments, "nginx")
-
-	remove(t, bundlePath)
-
-	// Test both podinfo and nginx deploy
-	deployPackagesFlag(bundlePath, "podinfo,nginx")
-	deployments, _, _ = e2e.UDS(cmd...)
-	require.Contains(t, deployments, "podinfo")
-	require.Contains(t, deployments, "nginx")
-
-	// Remove only podinfo
-	removePackagesFlag(bundlePath, "podinfo")
-	deployments, _, _ = e2e.UDS(cmd...)
-	require.NotContains(t, deployments, "podinfo")
-	require.Contains(t, deployments, "nginx")
-
-	// Remove nginx
-	removePackagesFlag(bundlePath, "nginx")
-	deployments, _, _ = e2e.UDS(cmd...)
-	require.NotContains(t, deployments, "podinfo")
-	require.NotContains(t, deployments, "nginx")
 
-	// Test invalid package deploy
-	_, stderr := deployPackagesFlag(bundlePath, "podinfo,nginx,peanuts")
-	require.Contains(t, stderr, "invalid zarf packages specified by --packages")
+	t.Run("Test only podinfo deploy (local pkg)", func(t *testing.T) {
+		deployPackagesFlag(bundlePath, "podinfo")
+		cmd := strings.Split("zarf tools kubectl get deployments -A -o=jsonpath='{.items[*].metadata.name}'", " ")
```

Comment:
this cmd can get moved up since it doesn't get changed and gets used by other tests in this function